### ISSUE TITLE
Question about Svelte component testing

### ISSUE THREAD

**Issue by shinokada** [K-Sato1995/svelte-vertical-timeline#1]
Your `package.json` has all testing library for jest. Do you need them when you run `playwright test`?

```
"@babel/core": "^7.17.8",
"@babel/preset-env": "^7.16.11",
"@babel/preset-typescript": "^7.16.7",
"@playwright/test": "^1.20.0",
...
"@testing-library/jest-dom": "^5.16.3",
"@testing-library/svelte": "^3.1.0",
"@types/jest": "^27.4.1",
...
```

**Comment by K-Sato1995** [comment]
@shinokada 

Thanks for the question!
If you want to just use playwrite, the other testing libraries are unnecessary😄

The reason I installed those is simply because I just wanted to try out different testings methods(playwrite and jest/svelte-testing-library) for svelte components to see which one feels right for me!

**Comment by shinokada** [comment]
Thanks. 
How do you find them? Which one is better?

**Comment by K-Sato1995** [comment]
> How do you find them?

Good old Googling led me to those two ways(Playwrite and svelte-testing-library)

> Which one is better?

The conclusion I came to was that I would use svelte-testing-library for unit-testing groups of components and use playwrite for EtoE testing.


> I'd like to add tests to [Flowbite-Svelte](https://flowbite-svelte.vercel.app/) UI components.

This looks awesome!!
I unfortunetly wrote only the bare minimum(not even) tests for this component library....

I think the MUI's tests for their timeline components can be a great example for ur use case. But they use enzyme which tests  components in a different way.)

https://github.com/mui/material-ui/blob/master/packages/mui-lab/src/Timeline/Timeline.test.tsx

**Comment by K-Sato1995** [comment]
@shinokada 

> How do you find them?

Good old Googling led me to those two ways(Playwrite and svelte-testing-library)

> Which one is better?

The conclusion I came to was that I would use svelte-testing-library for unit-testing groups of components and use playwrite for EtoE testing.


> I'd like to add tests to [Flowbite-Svelte](https://flowbite-svelte.vercel.app/) UI components.

This looks awesome!!(I stared the repo ⭐⭐)
I unfortunetly wrote only the bare minimum(not even) tests for this component library....

I think the MUI's tests for their timeline components can be a great example for ur use case. But they use enzyme which tests  components in a different way.

https://github.com/mui/material-ui/blob/master/packages/mui-lab/src/Timeline/Timeline.test.tsx
